apps/console/src/features/applications/components/wizard/oauth-protocol-settings-wizard-form.tsx

```diff
@@ -451,6 +451,8 @@ export const OauthProtocolSettingsWizardForm: FunctionComponent<OAuthProtocolSet
                                         getSubmit={ (submitFunction: (callback: (url?: string) => void) => void) => {
                                             submitUrl = submitFunction;
                                         } }
+                                        productName={ window["AppUtils"].getConfig().ui.productName }
```

Comment:
Shall we take the productName from the store instead?

Reply:
Fixed with commit 77ce737